core/src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -24,6 +25,10 @@
      */
     String getName();
 
+    default ImmutableSet<String> getAliases() {
```

Comment:
Actually, it's impossible to do this with an annotation for the NTPublish operation.
This is why: https://github.com/WPIRoboticsProjects/GRIP/pull/527/files#diff-108198640a1078f1b20a004632a786dbR45